abstract.tex

```diff
@@ -0,0 +1,14 @@
+\begin{abstract}
+
+The Vera C. Rubin Observatory will revolutionize all areas of astronomy over the next decade with its unique wide-fast-deep multi-color imaging survey, the Legacy Survey of Space and Time (LSST).
+LSST will produce approximately 20TB of raw data per night, which will be automatically processed by the LSST Science Pipelines 
+to generate science-ready data products -- processed images, catalogs and alerts. 
+The LSST Science Requirements Document (SRD) defines Key Performance Metrics (KPMs), scalar quantities that measure the quality and scientific fidelity of the LSST data products, together with specifications that represent the capability or accuracy required by the system to achieve the scientific goals. 
```

Comment:
I wonder if we should be so heavily emphasizing the KPMs. On one level, this is a very simple story that we can tell, and the SRD is an accessible user-facing document (much more so than the LSR, OSS, and DMSR). On the other hand, we are envisioning a system that computes a much wider range of metrics than the KPMs. For the purpose of this abstract though, keeping things simple may be advantageous.

Reply:
I agree with Keith - the KPMs are a (very important!) subset of the variety of metrics we can (and will) calculate. (But it's also fine to leave the abstract as it is, and address this point in the text later...)

Reply:
I am trying to rewrite this in a more general manner.

Reply:
Let's keep specific discussion on KPMs for the main text and be more general in the abstract - new version committed. Please re-review